src/Ipfs.Test/ClientTests.cs

```diff
@@ -31,5 +35,90 @@ public void ClientShouldThrowIfMethodCalledAfterBeingDisposed()
                 throw ex.InnerException;
             }
         }
+
+        [TestMethod]
+        public async Task ClientShouldBeAbleToDownloadLargeFiles()
+        {
+           /* This test is a bit long because it relies of having 
+            * a large file available through IPFS and the best way to
+            * ensure that is to simply create that file in the first place */
+            
+            var sourceFile = Path.GetTempFileName();
+            var targetFile = Path.GetTempFileName();
+
+            try
+            {
+                using (var client = new IpfsClient())
```

Comment:
This could be created with a Mock response like in this test: https://github.com/TrekDev/net-ipfs-api/blob/master/src/Ipfs.Test/ClientTests.cs#L25